src/4.3_to_5.0/_cleanupDrilldownLevel.ts

```diff
@@ -11,21 +15,34 @@ import { produce } from "immer";
 import _set from "lodash/set";
 
 /**
- * If the input set holds members of the leaf level, then DrilldownLevel can be replaced by it.
+ * A DrilldownLevel can be replaced by its first argument if both the following conditions are met:
+ * - It has a single argument
```

Comment:
```suggestion
 * - It has a single argument, which is a set.
```
Otherwise, the _"this set"_ is ambiguous on the next line.